app/src/main/kotlin/io/homeassistant/companion/android/sensors/GeocodeSensorManager.kt

```diff
@@ -70,42 +70,34 @@ class GeocodeSensorManager : SensorManager {
             return
         }
 
-        val location = try {
-            LocationServices.getFusedLocationProviderClient(context).lastLocation.await()
-        } catch (e: Exception) {
-            Timber.e(e, "Failed to get fused location provider client")
-            null
+        val location: Location? = getLocation(context)
```

Comment:
Unnecessary type hint

```suggestion
        val location = getLocation(context)
```